client/src/pages/Shop.js

```diff
@@ -151,115 +51,94 @@ export default function Shop() {
         });
         dispatch({
           type: "FETCH_SUCCESS",
-          payload: result.data.shop,
+          payload: result.data,
           user: userid
         });
       } catch (err) {
         dispatch({ type: "FETCH_FAILURE", error: err.message });
       }
     }
     dispatch({ type: "FETCH_INIT" });
-   
+
     fetchShop();
   }, []);
+  const UPDATE_DETAILS_API = "http://localhost:3001/shop/details-update/";
+  async function updateDetails() {
+    const config = {
+      headers: { "auth-token": token }
```

Comment:
did we ever create a utils to do these auth requests?

Reply:
Naresh created a request util but it was not yet merged when I began.